cmd/clusterctl/pkg/client/cluster/inventory.go

```diff
@@ -302,3 +286,92 @@ func (p *inventoryClient) GetDefaultProviderNamespace(provider string) (string,
 	// There is no provider or more than one namespace for this provider; in both cases, a default provider namespace cannot be decided.
 	return "", nil
 }
+
+func (p *inventoryClient) GetManagementGroups() ([]ManagementGroup, error) {
```

Comment:
I'd move this method and the relative functions to a different file

Reply:
done!